internal/server/repository_controller_test.go

```diff
@@ -14,6 +13,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	removeControllerSql = `DELETE FROM server_controller WHERE private_id = $1`
```

Comment:
```suggestion
	removeControllerSql = `delete from server_controller where private_id = $1`
```

Again, as per our sql style guide, use lower case for sql keywords.